### PR TITLE
Comments related to the iOS pull request of branch Justin/Spark_Release

### PR DESCRIPTION
**Pull request by LightningToast**

### CODE REVIEW

File: Classes/UI/SparkConnectingProgressViewController.m [hunk 1]
```diff
@@ -29,7 +29,7 @@
 NSInteger const kMaxRetriesConfigureAP = 5;
 NSInteger const kMaxRetriesConnectAP = 5;
 NSInteger const kMaxRetriesReachability = 5;
-NSInteger const kWaitForCloudConnectionTime = 1;
```
Comment (philipmeier) [inline Classes/UI/SparkConnectingProgressViewController.m:32]: I understand that Particle said 1, finger foods switched to 3.   And we might *keep* at 3.  Can you confirm what this  delay is in 1.0.9?

File: Classes/UI/SparkConnectingProgressViewController.m [hunk 2]
```diff
@@ -449,7 +449,7 @@ -(void)getDeviceAndFinishSetup
                 self.setupResult = SparkSetupMainControllerResultSuccess;
             }
             
-            dispatch_time_t popTime = dispatch_time(DISPATCH_TIME_NOW, 1 * NSEC_PER_SEC);
```
Comment (philipmeier) [inline Classes/UI/SparkConnectingProgressViewController.m:452]: I am ok with this, given that you tested it.  but i would rather put energy into  solution that blocks longer and looks good. and tolerates a skip of the waiting, after we confirm things are good and ready.   layer 1 is that google is reached.  layer 2 is that we hit a route on our cloud and confim hub is online.  potentially:   cloud check for any device report's presence in the last 5 mins.  We may need a new route.

File: Classes/UI/SparkConnectingProgressViewController.m (hunk 1 above)
Comment (LightningToast) [reply Classes/UI/SparkConnectingProgressViewController.m:32]: Delay in 1.0.9 was 1 second

File: Classes/UI/SparkConnectingProgressViewController.m (hunk 2 above)
Comment (LightningToast) [reply Classes/UI/SparkConnectingProgressViewController.m:452]: Should I change this to 2 then?

**Pull request merged**